python/mlcroissant/pyproject.toml

```diff
@@ -66,3 +66,8 @@ exclude = ["recipes*"]
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+markers = [
```

Comment:
According to the documentation (https://docs.pytest.org/en/latest/how-to/mark.html), you can also add: `addopts = --strict-markers`

Reply:
Done.